deploy.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# deploy.sh - Deployment script for News Search API and UI
+
+# Environment variables
+
+IMAGE_TAG="latest"  # Change this based on deployment (staging, production, v1.0)
+INDEXES="mc_search"
+ESHOSTS="http://ramos.angwin:9200,http://woodward.angwin:9200,http://bradley.angwin:9200"
+ESOPTS="{'timeout': 60, 'max_retries': 3}" # 'timeout' parameter is deprecated
+ELASTICSEARCH_INDEX_NAME_PREFIX="mc_search-*"
+TERMFIELDS="article_title,text_content"
+TERMAGGRS="top,significant,rare"
+APP_NAME="news-search-api"
+
+# Check if running as root
+if [[ $UID -ne 0 ]]; then
+    echo "This script must be run as root."
+    exit 1
+fi
+
+LOGIN_USER=$(whoami)
```

Comment:
`whoami` should always return `root` here since the above test has already weeded everyone else out, this is why I used `$(who am i | awk '{ print $1 }')`.  This doesn't work from a cron job -- c'est la vie!